qemu/tests/s390x_cpu_model_boot.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Boot guest with supported cpu-models
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    boot_cpu_models = params['boot_cpu_models'].split(';')
+    params['start_vm'] = 'yes'
```

Comment:
already exists in cfg, so I thought, no need here.

Reply:
it was needed cause I need to measure different cpu models to decide which should be booted

Reply:
solved